examples/examples/z_ping_shm.rs

```diff
@@ -15,7 +15,7 @@ use std::time::{Duration, Instant};
 
 use clap::Parser;
 use zenoh::{
-    buffers::ZSlice,
+    internal::buffers::ZSlice,
```

Comment:
I'm not sure it its normal to use internal API in examples

Reply:
https://github.com/eclipse-zenoh/zenoh/issues/1081